rpcserver.go

```diff
@@ -2957,13 +2957,85 @@ func (r *rpcServer) QueryAssetRoots(ctx context.Context,
 			"given universe")
 	}
 
-	issuanceRoot, err := r.cfg.UniverseArchive.RootNode(ctx, universeID)
+	// Query for both a issaunce and transfer universe root.
+	assetRoots, err := r.queryAssetProofRoots(ctx, universeID)
 	if err != nil {
+		return nil, err
+	}
+
```

Comment:
If we have an actual root here, then shouldn't we just return early rather than relying on the default behavior below?

Reply:
We can; in practice, `mayBeGrouped` would be false, so we skip the group lookup.

Looked it and early exit does reduce indention though, so switched to that.